scripts/build_avado.sh

```diff
@@ -49,6 +56,17 @@ sed -i "s/image:[ ]'hopr\.avado\.dnp\.dappnode\.eth:[0-9]\{1,\}\.[0-9]\{1,\}\.[0
 # Write dappnode version
 sed -i "s/\"version\":[ ]\"[0-9]\{1,\}\.[0-9]\{1,\}\.[0-9]\{1,\}\"/\"version\": \"${AVADO_VERSION}\"/" ./dappnode_package.json
 
+declare default_development_environment="master-goerli"
+
+if [[ -z $(grep -E "${default_development_environment}" "./build/Dockerfile") ]]; then
+  # Fail if default environment is no longer `master-goerli`
+  msg "Avado Dockerfile differs. Could not find \"${default_development_environment}\" environment in it"
+  exit 1
+fi
+
+# Overwrite default environmnet in Dockerfile with currently used one
+sed -i "s/master-goerli/${environment_id}/" ./build/Dockerfile
```

Comment:
You should undo this at the end to make this script produce less side-effects.

Reply:
Also, I think this will not work on macOS, because `-i` works differently on BSD sed. You might want to use a temporary output file.

Reply:
Done